app/helpers/sessions_helper.rb

```diff
@@ -10,6 +10,10 @@ module SessionsHelper
     "30 days" => 30.days.to_i
   }.freeze
 
+  # For security reasons we severely restrict the duration of impersonated
+  # sessions
+  IMPERSONATED_SESSION_DURATION = SESSION_DURATION_OPTIONS.fetch("1 hour")
```

Comment:
This is one hour instead of two (as we were discussing) because we still need it to be a valid value for `UserSession` (see #10660)

Reply:
1 hour sounds good to me!